plugins/modules/facter.py

```diff
@@ -52,6 +58,19 @@
         - timezone
         - is_virtual
 '''
+
+RETURN = r'''
+ansible_facts:
+  description: Dictionary with one key C(facter).
+  returned: when want_facts=true
+  type: dict
+  contains:
+    facter:
+      description: Dictionary containing facts discovered in the remote system.
+      returned: always
```

Comment:
This should be here:
```suggestion
      returned: when O(want_facts=true)
```
since the above `returned` isn't shown in the HTML docs.